internal/linter/check_enum.go

```diff
@@ -0,0 +1,35 @@
+package linter
```

Comment:
Please rename this file to check_has_enum.go, for consistency with other similar linter rules